src/test/cli/ceph-authtool/cap-invalid.t

```diff
@@ -3,10 +3,10 @@
 
 # TODO is this nice?
   $ ceph-authtool --cap osd 'broken' kring
-  $ ceph-authtool kring --list|grep -P '^\tcaps '
+  $ ceph-authtool kring --list|grep -E '^[^I]caps '
```

Comment:
On 20-7-2016 12:37, Kefu Chai wrote:
> In src/test/cli/ceph-authtool/cap-invalid.t
> <https://github.com/ceph/ceph/pull/10319#discussion_r71502243>:
> 
>> @@ -3,10 +3,10 @@
>>  
>>  # TODO is this nice?
>>    $ ceph-authtool --cap osd 'broken' kring
>> -  $ ceph-authtool kring --list|grep -P '^\tcaps '
>> +  $ ceph-authtool kring --list|grep -E '^[^I]caps '
> 
> i never knew that ERE supported the syntax like this! thought it's a
> character class excluding capital "I"! and seems the spec does not
> mention this usage, see
> http://pubs.opengroup.org/onlinepubs/000095399/basedefs/xbd_chap09.html#tag_09_03_05

It is one of the few versions that I actually got to work with grep -E
Took me a while to get something that works on all ends and pieces.
So I'm very curious what a teuthology runs will bring.